### PR TITLE
Feature: Reduce size of image

### PR DESCRIPTION
**Pull request by svx**
- Removes plone installer docs
- Remove all autoadded packages, which you can find with apt-mark showauto

### CODE REVIEW

File: 5.0/5.0.7/debian/Dockerfile [hunk 1]
```diff
@@ -37,11 +37,13 @@ RUN buildDeps="curl sudo python-setuptools python-dev build-essential libssl-dev
  && sudo -u plone bin/buildout \
  && chown -R plone:plone /plone /data \
  && rm -rf /Plone* \
- && SUDO_FORCE_REMOVE=yes apt-get purge -y --auto-remove $buildDeps \
+ && ADDED_PACKAGES=`apt-mark showauto` \
```
Comment (avoinea) [inline 5.0/5.0.7/debian/Dockerfile]: I don't see these packages within the final image:
```
$ docker run -it --rm --user root plone bash
$ apt purge --auto-remove apt-mark showauto

Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package apt-mark
E: Unable to locate package showauto
```

Comment (svx) [inline 5.0/5.0.7/debian/Dockerfile]: Oh I guess the name of the var is confusing, it is not about removing apt-mark, apt-mark show auto will check for 'third party' packages/dependencies  installed by the buildDeps and will remove those.

you can also do instead:

``SUDO_FORCE_REMOVE=yes apt-get remove --purge -y $buildDeps $(apt-mark showauto) \``

Comment (avoinea) [inline 5.0/5.0.7/debian/Dockerfile]: :) Oh, now I see. Maybe you can use `$()` syntax to avoid confusion ;)

**Comment by svx** [comment]
@avoinea are you OK if I merge or do you have any concerns ?

**Comment by avoinea** [comment]
@svx I forgot about it, sorry. Merged ;)

**Comment by svx** [comment]
no :)
Next PR is coming up :), just a moment ...

**Pull request merged**